packages/quantic/force-app/main/default/lwc/quanticUserActionsToggle/quanticUserActionsToggle.js

```diff
@@ -78,7 +78,7 @@ export default class QuanticUserActionsToggle extends LightningElement {
       options: {
         ticketCreationDate: this.ticketCreationDateTime,
         excludedCustomActions: this.excludedCustomActions?.length
```

Comment:
Is it the best way to check? Because you could call 
`Array.isArray()` ? 
Because for example a String will also have a `.length` ?

Reply:
I don't know if `Array.isArray` works with the Salesforce proxy object though 😂

Reply:
very nice point!
and yes Array.isArray works with the Salesforce proxy object.